networking/networking.go

```diff
@@ -33,6 +33,10 @@ const (
 	selfNetNS     = "/proc/self/ns/net"
 )
 
+var (
+	IPv4Loopback = net.ParseIP("127.0.0.1")
```

Comment:
Ah yes, that was from an intermediate version that also populated the env var for host mode.